### ISSUE TITLE
Show proper a11y issues in duplicate ID rule [3ea0c8]

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#1181]
The examples all use `div` elements, which are not known to cause accessibility problems. It would be better to have known accessibility problems as failure examples for this rule.

**Comment by Jym77** [comment]
Is this a duplicate of #1212 ?
